enhancements/single-node/aos-sno-pair.md

```diff
@@ -0,0 +1,143 @@
+  approvers:
+- TBD
+
+creation-date: 2021-07-12
+last-updated: 2021-07-12
+status: implementable
+---
+
+# Atomic OpenShift Single-Node-Pair
+
+## Release Signoff Checklist
+
+- [x] Enhancement is `implementable`
+- [ ] Design details are appropriately documented from clear requirements
+- [ ] Test plan is defined
+- [ ] Graduation criteria for dev preview, tech preview, GA
+- [ ] User-facing documentation is created in [openshift-docs](https://github.com/openshift/openshift-docs/)
+
+## Summary
+This enhancement does **not** propose replacing or otherwise modifying OpenShift's use of etcd.
+
+This enhancement proposes a layer on top of exactly two standard Single Node OpenShift clusters that monitors for node-level failures **and** uses existing OpenShift APIs to scale up Deployments once the peer has been put into a safe state.
+## Motivation
+
+Some companies have a need for a highly available container management solution that fits within a reduced footprint.
+- The hardware savings are significant for customers deploying many remote sites (kiosks, branch offices, restaurant chains, etc), most notably for edge computing and RAN specifically.
+- The physical constraints of some deployments prevents more than two nodes (planes, submarines, satellites, and also RAN).
+- Some locations will not have reliable network connections or limited bandwidth (once again submarines, satellites, and disaster areas such as after hurricanes or floods)
+  
+### Goals
+
+Provide the ability for exactly two all-in-one OpenShift clusters to operate as a **predefined** pair in an **active-active** configuration, detect when its peer has died and **automatically** take over its workloads after ensuring it is safe to do so.
+
+### Non-Goals
+
+- Management as a single entity (pod/namespace replication).
+- **Replication of application data** between appliances.
+- DR for services in clusters with **more than a single node.**
+- DR for services running on **two or more all-in-one clusters.**
+- Enable services uptime during a **remote cluster upgrade process.**
+
+## Proposal
+
+Assuming workloads that require some form of coordination between the two nodes (either at-most-one semantics or simply to reduce machine load during normal operation):
+
+- Pacemaker and Corosync from the RHEL-HA cluster stack are deployed [in Pods](https://github.com/beekhof/installer/blob/aio/pod-cluster.yaml.in) on both OCP clusters, and they are configured to form a RHEL-HA cluster which is composed of two independent SNO clusters.
+- The Pacemaker cluster is configured so that on peer failure the cluster can reliably ensure that the workload on an unhealthy peer is stopped before starting the workload on the healthy peer. Configuration [flexibility](https://github.com/openshift/enhancements/blob/fb3faa767a04866ee6d6ef46127636bd086df658/enhancements/single-node/aos-sno-resources/2%20Node%20HA%20Menu.svg) is provided here to allow for the anticipated variety of customer requirements.
```

Comment:
I think the URL can just point to the local file and rely on relative URL resolution. Otherwise every time we update the image we have to update  the link, too.